Source/Fuse.Elements/Resources/MultiDensityImageSource.uno

```diff
@@ -93,6 +94,18 @@ namespace Fuse.Resources
 			get { return _active; }
 		}
 
+		public override ImageOrientation Orientation
+		{
+			get
+			{
+				if (_active != null)
+				{
+					return _active.Orientation;
+				}
+				return ImageOrientation.Identity;
```

Comment:
Why isn't this just `return _proxy.Orientation;`, similar to `Size` and `PixelSize`?

Reply:
Just because I didn't think it through well enough I think. I'll change it to match the way Size and PixelSize works.